### DISCUSSION TITLE
Slider tick/end columns should be the same as the max combo column

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19194] [Q&A] [answered]
Current look is kinda out of style with the max combo column. I think you should highlight the maximum using green color as its done with the max combo column. The same thing with drum tick, droplets and hold ticks.
![image](https://user-images.githubusercontent.com/86934170/179405782-54f61a66-4ede-45b0-b5ab-6dd92417b5a0.png)

**Comment by frenzibyte** [comment]
Implementing this is quite simple, but I'm not 100% sure if it looks correct to have the same colour as a stat which was perfected like accuracy and combo (most scores are likely to have full slider ticks/ends):

<img width="1332" alt="CleanShot 2022-07-18 at 03 17 012" src="https://user-images.githubusercontent.com/22781491/179430691-2d304f69-275f-4131-b6f5-4db3da97ee4b.png">

@peppy asking for your opinion before I PR this.

**Reply by Loreos7** [reply]
No, I meant that you have to hide the maximum of **possible** ticks. For example:

![image](https://user-images.githubusercontent.com/86934170/179472021-8b4648a9-0add-4133-b112-a96dc674b789.png)

Just 83
Then 82
Then 26 Highlighted
Then 26 Highlighted
Then 84 Highlighted
All of that would mean with classic mode the maximum is 84 (because its highlighted), without classic mode its 26. I think the more highlighted, the better.
But at least for now its just hard to read this amount of numbers. 

Just imagine what it would be if the max combo had the same style. (1324x/2594x)

**Reply by peppy** [reply]
I agree with hiding the max and also highlighting the case where 100% are reached. It feels like too much (repeated) information.

We can probably show the fractional version when hovering as a tooltip?

**Comment by frenzibyte** [comment, marked as answer]
Will be addressed in https://github.com/ppy/osu/pull/19244
